Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -127,7 +127,10 @@ final class Navigator: CoreNavigator {
     
     private static weak var _navigator: Navigator?
     // Used in tests to recreate the navigator
-    static func _recreateNavigator() { _navigator = nil }
+    static func _recreateNavigator() {
```

Comment:
We might not need this method anymore, it used to be helpful, but now `_navigator` is marked as `weak`, so it should recreate automatically in each test

Reply:
Thanks, removed this method